commands/make/make.drush.inc

```diff
@@ -281,6 +289,31 @@ function drush_make($makefile = NULL, $build_path = NULL) {
 }
 
 /**
+ * Drush callback; convert makefile to YAML.
+ */
+function drush_make_convert_yaml($makefile = NULL) {
```

Comment:
Presumably the input makefile is required. So why provide an argument default here?